xcp_d/tests/test_workflows_anatomical.py

```diff
@@ -1,18 +1,135 @@
-"""Tests for the xcp_d.workflows.anatomical module."""
+"""Tests for the xcp_d.workflow.anatomical module."""
```

Comment:
should we change this to `workflows`?

Reply:
Good catch. Not sure how I managed to revert that.